src/types/timezone.jl

```diff
@@ -1,28 +1,38 @@
-const TIME_ZONES = Dict{String,TimeZone}()
+const TIME_ZONE_CACHE = Dict{String,Tuple{TimeZone,Class}}()
 
 """
-    TimeZone(str::AbstractString) -> TimeZone
+    TimeZone(str::AbstractString, mask::Class=Class.DEFAULT) -> TimeZone
 
-Constructs a `TimeZone` subtype based upon the string. If the string is a recognized time
-zone name then data is loaded from the compiled IANA time zone database. Otherwise the
-string is assumed to be a static time zone.
+Constructs a `TimeZone` subtype based upon the string and the provided `mask`. If the
```

Comment:
I think this over-emphasizes `mask`. Most people won't ever need to touch it. Maybe list `TimeZone(str)` first at the top and note that it defaults to `DEFAULT`, then list `TimeZone(str, mask)`?

Reply:
I can make these separate docstrings (still one function definition) to hide most of the mask details